lib/gitlab/ci/templates/jobs/gradle/PublishJar.yml

```diff
@@ -1,5 +1,6 @@
 variables:
   GIT_TASKS_ENABLED: "true"
+  SKIP_PUBLISH_TESTS: "true"
```

Comment:
Related to https://github.com/chesapeaketechnology/gitlab-templates/pull/140#issuecomment-3390939514,  please set SKIP_PUBLISH_TESTS as "false" for release/3.x.x so this isn't a breaking change.